wdio.conf.ts

```diff
@@ -0,0 +1,288 @@
+import { Config } from "webdriverio";
```

Comment:
sync = true and debug variable have been removed.

1. With sync no longer as true will this cause any parallelism issues when tests are being executed
2. With the debug variable removed to listen to the debugger, have we confirmed that tests can be debugged in VS via the play button?